config/app.php

```diff
@@ -92,6 +92,7 @@
      */
 
     'locale' => env('APP_LOCALE', 'en'),
+    'default_locale' => env('APP_LOCALE', 'en'), // just a holder for the system set locale
```

Comment:
i dont understand what this does since its always the same as `locale`

Reply:
app.locale gets set at runtime to the user's locale... open to better solutions.